apps/api/src/beneficiary/beneficiary.service.ts

```diff
@@ -41,4 +41,10 @@ export class BeneficiaryService {
     if (!result) throw new NotFoundException('sorry id not found')
     return result
   }
+
+  async removeManyBeneficiaries(ids: string[]) {
+    return ids.map(async (x) => {
+      return await this.removeBeneficiary(x)
+    })
+  }
```

Comment:
This code is sending queries to the database in a loop that is not optimal and contradicts the best practices.

A much more leaner approach would be to collect all ids and do a `deleteMany` query with `in: ids` clause.

It will end up as a single query deleting many records at once.

Reply:
Done.